stl/inc/mutex

```diff
@@ -463,7 +463,7 @@ public:
     }
 
     ~scoped_lock() noexcept {
-        _For_each_tuple_element(_MyMutexes, [](auto& _Mutex) noexcept { _Mutex.unlock(); });
+        apply([](_Mutexes&... _Mtxes) { (..., void(_Mtxes.unlock())); }, _MyMutexes);

```

Comment:
Needs _STD

Reply:
What's the rationale behind _STD? I will change and update PR.

Reply:
To prevent ADL -- that is to say, we don't want to find some function the user declared named apply rather than `::std::apply`.